### PR TITLE
it is 5 lines :D

### PR DESCRIPTION
**Pull request by sevki**

### CODE REVIEW

**Comment by bldybuild** [comment]
Commit 50c8e8c1634a38a3252eee25fd7a671a8f478190 is not signed-off.

Please fix these issues:

>	1. Sevki has not signed-off 50c8e8c.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

File: cmd/ufs/ufs.go [hunk 1]
```diff
@@ -30,7 +36,7 @@ func main() {
 		l.Trace = nil // log.Printf
 		return nil
 	})
-
+
```
Comment (rminnich) [inline cmd/ufs/ufs.go:39]: fix the white space?

File: cmd/ufs/ufs.go [hunk 2]
```diff
@@ -16,11 +16,17 @@ import (
 var (
 	ntype = flag.String("ntype", "tcp4", "Default network type")
 	naddr = flag.String("addr", ":5640", "Network address")
+	haddr = flag.String("http", ":8080", "Network address")
+	root  = flag.String("root", "/", "Set the root for all attaches")
```
Comment (rminnich) [inline cmd/ufs/ufs.go:20]: Do we want a root for http and a root for attaches?

Comment (sevki) [reply cmd/ufs/ufs.go:20]: I was thinking of using the [attach root]( https://github.com/Harvey-OS/ninep/blob/8010cab8eac948c00149d228e8043bb950d124d6/filesystem/filesystem.go#L45)  as the http root. Should I just parse that flag once and pass it in to newUFS as an arg on line :35?

Comment (rminnich) [reply cmd/ufs/ufs.go:20]: it's confusing as worded for me. Attaches sounds like 9p.

"set the root for http service"
?

**Comment by sevki** [comment]
rolled in the changes from #37

**Comment by sevki** [comment]
closing this since we're switching to p9

**Pull request closed without merging**